src/blueprint/README.md

```diff
@@ -0,0 +1,153 @@
+# Azure CLI blueprint Extension #
+This package is for the 'blueprint' extension, i.e. 'az blueprint'. More info on what is [Blueprint](https://docs.microsoft.com/azure/governance/blueprints/overview).
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name blueprint
+```
+
+Blueprint can be scoped in a subscription or management group, which is a group of subscriptions. You can use management group by specify `--management-group`, or use subscription by specify `--subscription.` If both parameters are omitted, the command will use your default subscription. You can set it by running:
+```
+az account set --subscription subscription_id
+```
+
+### Included Features
+#### Blueprint Definition Management:
+*Examples:*
+
+##### Import Blueprint Definition and Artifacts Settings
+
+```
+az blueprint import \
+--name blueprintName \
+--input-path "/path/to/blueprint/directory"
+
+```
+
+In the input directory, there should be a file named "blueprint.json" that has blueprint definition and parameters for artifacts. There should be a subdirectory named "artifacts" and it should contain files for artifact definition. Examples can be found [here]().
```

Comment:
All singular nouns should have an article in front of them. So, "...that has blueprint definition" should be either: "...that has **a** blueprint definition" or if multiple definitions are allowed, plural, then it should be something like: "...that has blueprint definition**s**". The same grammatical error occurs in the next sentence as well: "...files for artifact definition" should have an article in front of the noun as well: "files for **an** artifact definition"

Reply:
updated